seatunnel-engine/seatunnel-engine-server/src/test/java/org/apache/seatunnel/engine/server/CoordinatorServiceTest.java

```diff
@@ -154,7 +154,7 @@ public void testJobRestoreWhenMasterNodeSwitch() throws InterruptedException {
             });
 
         // wait job restore
-        Thread.sleep(5000);
+        Thread.sleep(30000);
```

Comment:
This is not a good practice. @Hisoka-X do you have any good ideas?

Reply:
If we can catch log to make sure job restore success, maybe we can remove this code. But now, we don't have good way.

Reply:
Agree with you, I think the problem is the code getJobMaster(jobId), which returns null when the master has not successfully switched.
If we can't make sure job restore success now, we can either temporarily discard it or set the code Thread.sleep to a large value.

Reply:
`Thread.sleep(5000) ` is used to wait the job begin restore and job status turn to a non `RUNNING` status. And then I use 

```
// pipeline will recovery running state
        await().atMost(200000, TimeUnit.MILLISECONDS)
            .untilAsserted(
                () -> Assertions.assertEquals(PipelineStatus.RUNNING,
                    server2.getCoordinatorService().getJobMaster(jobId).getPhysicalPlan().getPipelineList().get(0)
                        .getPipelineState()));

```

to ensure the job restore complete and job status turn to running again.